### PR TITLE
Document why we don't have a race condition removing v2v

### PR DESCRIPTION
**Pull request by tiraboschi**
The removal of vm-import-operator is tricky
with its deployment removed just as a consequence
of the removal of the old CSV.
Let's document it to prevent introducing regressions
in the future.

Signed-off-by: Simone Tiraboschi <stirabos@redhat.com>

**Reviewer Checklist**

> Reviewers are supposed to review the PR for every aspect below one by one. To check an item means the PR is either "OK" or "Not Applicable" in terms of that item. All items are supposed to be checked before merging a PR. 

- [ ] PR Message
- [ ] Commit Messages
- [ ] How to test
- [ ] Unit Tests
- [ ] Functional Tests
- [ ] User Documentation
- [ ] Developer Documentation
- [ ] Upgrade Scenario
- [ ] Uninstallation Scenario
- [ ] Backward Compatibility
- [ ] Troubleshooting Friendly
  
**Release note**:
```release-note
NONE
```

### CODE REVIEW

**Comment by tiraboschi** [comment]
let's carefully double check this before merging
/hold

**Review by nunnatsa — APPROVED** [review]
/approve

**Pull request merged**
